systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -289,51 +290,62 @@ def post_configure(self, address):
         route = CsRoute()
         if not self.get_type() in ["control"]:
             route.add_table(self.dev)
-            
             CsRule(self.dev).addMark()
```

Comment:
addMark should be not applied to eth0, so we can move set_mark togeterh with addMark. like
```
            if self.dev != 'eth0':
                CsRule(self.dev).addMark()
                self.set_mark()
```